src/shared/proto/robot_component_status.proto

```diff
@@ -0,0 +1,132 @@
+syntax = "proto2";
+
+/* Data about the status of the break beam */
+message BreakBeamStatus
+{
+    // Whether or not the ball is interrupting the robot’s laser beam.
+    required bool ball_in_beam = 1;
+
+    // The reading of the robot’s laser sensor.
```

Comment:
Can we add units / a range to this value?

Reply:
done